### ISSUE TITLE
Allow switch back and forth from editor mode

### ISSUE THREAD

**Issue by RiscadoA** [GameDevTecnico/cubos#1011]
Blocked by #464.

# Problem
With editor mode implemented, we still aren't able to, for example, start the game by pressing a button in the editor.
It should also be possible to reset the game from within the editor, without having to restart the editor.

# Solution
It's not clear yet how this would be implemented, but the big challenges are:
- how to run all of the game startup systems **only** when the play button is pressed.
- how to restore the World state to what it was before the game started playing.
- how to run the game startup systems **again** if we had a sequence like play -> reset -> play.

**Comment by RiscadoA** [comment]
No longer fits with the new plan we have for the editor architecture.
